integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/query/SearchQueryTimeoutIT.java

```diff
@@ -118,7 +118,7 @@ public void timeout_slowQuery_smallTimeout_limitFetching() {
 				.truncateAfter( 1, TimeUnit.NANOSECONDS )
 				.fetchAll();
 
-		assertThat( result.totalHitCount() ).isLessThan( TOTAL_DOCUMENT_COUNT );
+		assertThat( result.totalHitCount() ).isNull();
```

Comment:
That's... surprising. What's going on here?

Reply:
Already fixed